service/src/main/resources/templates/ehr_extract_template.mustache

```diff
@@ -0,0 +1,57 @@
+<EhrExtract classCode="EXTRACT" moodCode="EVN">
+    <id root="{{ehrExtractId}}" />
+    <statusCode code="COMPLETE" />
+    <availabilityTime value="{{availabilityTime}}" />
+    <recordTarget typeCode="RCT">
+        <patient classCode="PAT">
+            <id root="2.16.840.1.113883.2.1.4.1" extension="{{patientId}}" />
+        </patient>
+    </recordTarget>
+    <author typeCode="AUT">
+        <time value="{{authorTime}}" />
+        <AgentOrgSDS classCode="AGNT">
+            <agentOrganizationSDS classCode="ORG" determinerCode="INSTANCE">
+                <id root="1.2.826.0.1285.0.1.10" extension="{{destinationId}}" />
+            </agentOrganizationSDS>
+        </AgentOrgSDS>
+    </author>
+    <destination typeCode="DST">
+        <AgentOrgSDS classCode="AGNT">
+            <agentOrganizationSDS classCode="ORG" determinerCode="INSTANCE">
+                <id root="1.2.826.0.1285.0.1.10" extension="{{authorId}}" />
```

Comment:
This should be named `destinationId` and the value should be the fromOdsCode from the EhrRequest